pandas/tests/groupby/test_groupby.py

```diff
@@ -1297,6 +1297,36 @@ def test_groupby_nat_exclude():
             grouped.get_group(pd.NaT)
 
 
+def test_groupby_two_group_keys_all_nan():
+    # Grouping over two group keys all nan raised an error previously
```

Comment:
This comment only makes sense during this PR; can replace with "...all nan shouldn't raise". Also - if there is no  github issue, label with this PR #

Reply:
Thanks, that makes more sense.